pkg/collect/run_pod.go

```diff
@@ -48,6 +50,13 @@ func (c *CollectRunPod) Collect(progressChan chan<- interface{}) (CollectorResul
 		return nil, errors.Wrap(err, "failed to create client from config")
 	}
 
+	if c.Collector.CollectorName == "" {
+		c.Collector.CollectorName = "run-pod"
+		if c.Collector.Name != "" {
+			c.Collector.CollectorName = c.Collector.Name
+		}
+	}
+
```

Comment:
@banjoh @xavpaice And last thing, take a look at the updates in my last commit. On [troubleshoot.sh](https://troubleshoot.sh/docs/collect/run-pod/) for the `runPod` collector we document the output path to be `collector-name/collector-name.log`, but that hasn't been the case for likely a very long time due to a bug in whether or not the user specifies `collectorName` or `name` in the spec. This PR includes the changes to fix that, but it may break what has been expected behavior in `runPod` files being left at the root of the bundle. So we either need to leave as is and correct the documentation, or fix it. I lean towards the latter.

Reply:
We will need to document the change as breaking by fixing something that was already broken, I guess :)